numismatic/cli.py

```diff
@@ -66,10 +68,10 @@ def coin(state, feed, cache_dir, requester, log_level):
         coin listen -a BTC,ETH,XMR,ZEC collect -t Trade run -t 30
     '''
     logging.basicConfig(level=getattr(logging, log_level.upper()))
-    from .datafeeds import Datafeed
+    
     state['cache_dir'] = cache_dir
     state['datafeed'] = \
-        Datafeed.factory(feed_name=feed, cache_dir=cache_dir,
+        Feed.factory(feed_name=feed, cache_dir=cache_dir,
```

Comment:
If one could import into a namespace, then I would be able to say feed.base.factory...